src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/EnumCompletionProviderTests.vb

```diff
@@ -374,23 +374,6 @@ End Class
             Await VerifyNoItemsExistAsync(markup)
         End Function
 
-        <Fact, WorkItem(815963, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/815963")>
```

Comment:
This test is moved to `VisualBasicCompletionCommandHandlerTests.vb` since it now relies on SymbolCompletionProvider to suggest the local